referral.go

```diff
@@ -11,6 +11,15 @@ import (
 	"strings"
 )
 
+// A PaymentRefund that has the refund details for the refund request.
+type BetaPaymentRefund struct {
+	RefundedAmount          int        `json:"refundedamount,omitempty"`
+	RefundedPaymentLogs     []string   `json:"refundedpaymentlog,omitempty"`
+	PaymentLogId            string     `json:"paymentlogid,omitempty"`
+	RefundedAmountCurrencys string     `json:"refundedamountcurrencys,omitempty"`
```

Comment:
These all need to be snake case json elements. 
```suggestion
	RefundedAmount          int        `json:"refunded_amount,omitempty"`
	RefundedPaymentLogs     []string   `json:"refunded_payment_log,omitempty"`
	PaymentLogId            string     `json:"payment_log_id,omitempty"`
	RefundedAmountCurrencys string     `json:"refunded_amount_currencys,omitempty"`
```